src/Runtime/PyExecutionSessionBase.cpp

```diff
@@ -55,8 +55,8 @@ struct npy_format_descriptor<onnx_mlir::float_16> {
 namespace onnx_mlir {
 
 PyExecutionSessionBase::PyExecutionSessionBase(
-    std::string sharedLibPath, bool defaultEntryPoint)
-    : onnx_mlir::ExecutionSession(sharedLibPath, defaultEntryPoint) {}
+    std::string sharedLibPath, std::string tag, bool defaultEntryPoint)
+    : onnx_mlir::ExecutionSession(sharedLibPath, tag, defaultEntryPoint) {}
```

Comment:
This code will change significantly in #2373, my recommendation will be to find out the tags from the compile option strings.

Reply:
Sure, will update this to get from the compiler options once #2373 is landed.

Reply:
Done.